app/services/schools_in_area.rb

```diff
@@ -0,0 +1,45 @@
+class SchoolsInArea
+  attr_accessor :lat, :lng, :radius
```

Comment:
Should this be `attr_reader` since we're memoizing based on these values, so if they change, our data will become invalid?

Reply:
Good catch 👍